docs/learn/lagandpacketloss.md

```diff
@@ -1,109 +1,56 @@
 ---
 id: lagandpacketloss
-title: Latency and packet loss
+title: Understanding latency
 ---
 import ImageSwitcher from '@site/src/ImageSwitcher.js';
 
-A multiplayer game operating over the internet has to deal with several adverse factors that aren't present when developing single-player or LAN-only multiplayer games.
+Multiplayer games operating over the internet have to manage adverse network factors that don't affect single-player or LAN-only multiplayer games, most notably [network latency](#network-latency). Latency (also known as lag) is the delay between a user taking an action and seeing the expected result. When latency is too high, a game feels unresponsive and slow.
 
-:::important
- According to wikipedia, 200 ms of input lag is the lower threshold at which people typically notice lag.
+Generally, 200ms of latency is the point at which users notice gameplay degradation, although different types of games can tolerate more or less latency. For example, first-person shooter games perform best with less than 100ms of latency, whereas real-time strategy games can tolerate higher latency values of up to 500ms before gameplay is meaningfully impacted.
```

Comment:
nicely done.

Reply:
Thank 🙏 We do not reference Wikipedia in our homework.